OemPkg/OemConfigPolicyCreatorPei/OemConfigPolicyCreatorPei.c

```diff
@@ -0,0 +1,256 @@
+/** @file
+  This module receives the static platform data for a platform and creates a config policy. It is intended to run
+  before a platform level Silicon Policy Creator that maps this config policy to silicon policy.
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#include <Uefi.h>
+
+#include <Ppi/ReadOnlyVariable2.h>
+#include <Ppi/Policy.h>
+#include <Library/DebugLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/PeiServicesLib.h>
+#include <Library/DebugLib.h>
+#include <Library/MemoryAllocationLib.h>
+#include <Library/ConfigVariableListLib.h>
+
+typedef enum {
+  KNOB_PowerOnPort0,
+  KNOB_MAX
+} KNOB;

```

Comment:
I assume this will become UINTN based later?

Reply:
This was missing a commit, fixed now.